orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/rollback/ExplicitRollback.groovy

```diff
@@ -49,6 +51,9 @@ class ExplicitRollback implements Rollback {
   Boolean disableOnly
   Boolean enableAndDisableOnly
 
+  @Autowired
```

Comment:
I guess we should add the `@JsonIgnore` too

Reply:
Done